src/tests/modules/perl/test.pl

```diff
@@ -67,14 +67,14 @@ sub authenticate {
 	if ($RAD_REQUEST{'User-Name'} =~ /^baduser/i) {
 		# Reject user and tell him why
 		$RAD_REPLY{'Reply-Message'} = "Denied access by rlm_perl function";
-		return RLM_MODULE_REJECT;
+		return RLM_MODULE_OK;
```

Comment:
hm?

Reply:
For the purpose of the test this was needed - returning RLM_MODULE_REJECT causes the test to stop immediately.  I'll change it to RLM_MODULE_NOTFOUND and add a comment